latex2sympy.py

```diff
@@ -477,6 +472,34 @@ def convert_atom(atom):
             return sr
         except (TypeError, ValueError):
             return sympy.Number(s)
+    elif atom.SCI_NOTATION_NUMBER():
+        s = atom.SCI_NOTATION_NUMBER().getText()
+        s_parts = s.split('\\times 10^')
+        s1 = s_parts[0].replace(',', '')
+        try:
+            n1 = sympy.Rational(s1)
+        except (TypeError, ValueError):
+            n1 = sympy.Number(s1)
+        s2 = s_parts[1].replace('{', '').replace(',', '').replace('}', '')
+        try:
+            n2 = sympy.Rational(s2)
+        except (TypeError, ValueError):
+            n2 = sympy.Number(s2)
+        n_exp = sympy.Mul(n1, sympy.Pow(10, n2))
+        try:
+            n = sympy.Rational(n_exp)
+        except (TypeError, ValueError):
+            n = sympy.Number(n_exp)
+        return n
+    elif atom.FRACTION_NUMBER():
+        s = atom.FRACTION_NUMBER().getText().replace("\\frac{", "").replace("}{", "/").replace("}", "").replace(",", "")
+        try:
+            sr = sympy.Rational(s)
+            return sr
+        except ZeroDivisionError:
+            return sympy.S.EmptySet
```

Comment:
@builtbywill I'm testing some expressions and `1+1/0` is giving me an error `AttributeError: 'EmptySet' object has no attribute 'as_coeff_Mul'`. I'm wondering if returning empty set in the middle of the equation here is the cause, but I'm not sure.

Reply:
@tragessere oh nice catch, I will take a look

Reply:
@tragessere it looks like the error shows up only when trying to convert the sympy expression to string using `print` or `srepr`

it is happening on develop as well though, and even with the explicit `1+\emptyset` latex

Reply:
In other words I think you found an existing bug 😂

Reply:
Sounds good then, we can go ahead and make an issue for that I guess.